services_and_applications/slate/access_olcf_resources/mount_fs.rst

```diff
@@ -10,8 +10,10 @@ to get the necessary configuration injected into the container at runtime.
   :header: "Cluster", "Annotation", "Value", "Mounts"
   :widths: 5, 8, 5, 25
 
-  "Marble", "ccs.ornl.gov/fs", "olcf", "/ccs/sw, /ccs/home, /ccs/proj, /gpfs/alpine"
+  "Marble", "ccs.ornl.gov/fs", "olcf", "/ccs/sw, /ccs/home, /ccs/sys, /ccs/proj, /gpfs/alpine"
+  "Marble", "ccs.ornl.gov/fs", "olcf-nfs", "/ccs/sw, /ccs/home, /ccs/sys, /ccs/proj"
```

Comment:
We mount /gpfs/alpine with olcf-nfs

Reply:
Cool, do we also mount (or do we plan to mount) /gpfs/wolf with open-nfs?

Reply:
We don't right now, no plans to do so yet